backend/server/src/service/organisation.rs

```diff
@@ -0,0 +1,233 @@
+use chrono::{DateTime, Utc};
+use snowflake::SnowflakeIdGenerator;
+use sqlx::{Pool, Postgres};
+use uuid::Uuid;
+use crate::models::organisation::{Member, MemberList, OrganisationDetails, OrganisationRole};
+use crate::models::campaign::Campaign;
+use crate::models::error::ChaosError;
+
+pub async fn is_admin(user_id: i64, organisation_id: i64, pool: &Pool<Postgres>) -> Result<(), ChaosError> {
+    let is_admin = sqlx::query!(
+        "SELECT EXISTS(SELECT 1 FROM organisation_members WHERE organisation_id = $1 AND user_id = $2 AND role = 'Admin')",
+        organisation_id,
+        user_id
+    )
+        .fetch_one(pool)
+        .await?.exists.unwrap();
```

Comment:
What is the `.exists.unwrap()`? I guess we should try to use as little unwraps as possible. If you're sure it will never fail, using `expect("<message explaining why this is safe>")` is better practice and more readable.